contracts/test/psm/PegStabilityModuleNonCustodial.t.sol

```diff
@@ -0,0 +1,235 @@
+pragma solidity ^0.8.4;
+
+import {ERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import {MockPCVDepositV2} from "../../mock/MockPCVDepositV2.sol";
+import {IPCVDeposit} from "../../pcv/IPCVDeposit.sol";
+import {MockERC20} from "../../mock/MockERC20.sol";
+import {MockOracle} from "../../mock/MockOracle.sol";
+import {ICore} from "../../core/ICore.sol";
+import {Core} from "../../core/Core.sol";
+import {IFei, Fei} from "../../fei/Fei.sol";
+import {PegStabilityModuleNonCustodial} from "./../../peg/PegStabilityModuleNonCustodial.sol";
+import {PegStabilityModule} from "./../../peg/PegStabilityModule.sol";
+import {Vm} from "./../utils/Vm.sol";
+import {DSTest} from "./../utils/DSTest.sol";
+import {getCore, getAddresses, FeiTestAddresses} from "./../utils/Fixtures.sol";
+
+contract PegStabilityModuleNonCustodialTest is DSTest {
+
+    PegStabilityModuleNonCustodial private psm;
+    ICore private core;
+    IFei private fei;
+
+    modifier prank(address toPrank) {
+      vm.startPrank(toPrank);
+
+      _;
+
+      vm.stopPrank();
+    }
+
+    uint256 immutable public mintAmount = 10_000_000e18;
```

Comment:
Nitpick, but these `immutable` variables can become `constant` variables. You won't be changing the variables of these variables in the constructor. 

Not important though, given these are just tests...